composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": ">=2.0,<3.0",
+        "symfony/framework-bundle": ">=2.0,<=4.0",
```

Comment:
I would agree (and it is much clearer). Mostly was just following the previous style of specifying. Any objects to using this new version constraint?